scripts/extract-elements.js

```diff
@@ -6,13 +6,17 @@ const request = require('request');
 
 const refUrl = 'https://developer.mozilla.org/en-US/docs/Web/HTML/Element';
 const dataPath = './data/elements.txt';
+const expectedElCount = 131;
 
 /**
  * From the MDN HTML elements reference, extract a list of elements.
  */
 function extractElements($) {
     const excludeElements = [
         'html', 'head', 'body', 'style', 'h1–h6', 'input',
+        // out of scope, different namespaces - but Mozilla added these to the
+        // above reference page Jan 2021 so we need to exclude them now.
```

Comment:
perhaps reference the MDN PR that changed this for posterity? https://github.com/mdn/content/pull/410

Reply:
Ah nice, I didn't think to look for where this was introduced!